### DISCUSSION TITLE
A web-based tldr-pages client with multiple language support

### DISCUSSION THREAD

**Discussion by reinhart1010** [tldr-pages/tldr#7556] [Show and tell]
Hi, I'm currently working on another web-based tldr-pages client with **support for multiple languages**. It is also based on [another Jekyll-based static wiki theme](https://github.com/reinhart1010/HAM) which I have made before.

For example: http://nix.reinhart1010.id/pt_BR/common/7z.html

![image](https://user-images.githubusercontent.com/17312341/146660328-687340f9-87ba-4f5b-96cb-7184ebf4d493.png)

![image](https://user-images.githubusercontent.com/17312341/146660399-fd73fb58-f8fd-4373-8a50-97962ea65d0c.png)

What do you guys think about it?

**Comment by sbrl** [comment]
Wow, that looks *awesome*! I assume you've been following the client spec? Don't hesitate to get in touch if you encounter any issues with it.

We've actually been having a small issue with the lack of maintenance of our current official client, so more web clients are certainly very welcome :D

Depending on how things pan out and what other maintainers say, if you've up for it we may be able to discuss making yours the  official one at some point.
